### ISSUE TITLE
Screen reader compatibility with Jaws and accessibility issues

### ISSUE THREAD

**Issue by jlevesque** [grommet/grommet#1776]
I have started to test my app with JAWS screen reader for Windows (https://www.freedomscientific.com/Downloads/JAWS ). JAWS is really popular among Windows users. I've noticed that there is some issue with some Grommet's component with JAWS. 

For instance, the dropdown are not readable by JAWS. The `<ol>` and `<li>` need a role comboList and option. The select component is not render as a `<select>` and `<option>` component, so JAWS reads it as a html list (`<ol>`). Therefore JAWS is enable to read the options and the options are not accessible with the keyboard. 

It's seems that some component have a `[Object object]` aria-label when this prop is not a string. 

Since Grommet is presented as an accessible friendly library, I think those issues should be look at. Also, doing tests with JAWS and other screen readers should be considered. Mac has there own voice over, but it doesn't work like other sreen reader sadly.

Dropdowns that causes issues (the one I have noticed so far): 
Menu component
Select component
TextInput's suggestion 

### Your Environment
* Grommet version: 1.9.0
* JAWS 2018

**Comment by L0ZZI** [comment]
Great catch. We'll investigate further to test the implementation but if you see an area to contribute we welcome Pull Requests from the community.
